internal/pkg/cli/root_test.go

```diff
@@ -151,7 +151,11 @@ func TestGetLogFileTempFile(t *testing.T) {
 	file, err := root.getLogFile()
 	assert.NoError(t, err)
 	assert.NotNil(t, file)
-	assert.True(t, strings.HasPrefix(root.options.LogFilePath, os.TempDir()+"/"))
+
+	// Linux returns temp dir without last separator, MacOs with last separator.
+	// ... so we need to make sure there is only one separator at the end.
+	tempDir := strings.TrimRight(os.TempDir(), string(os.PathSeparator)) + string(os.PathSeparator)
```

Comment:
Linux vracia ako vysledok `os.TempDir()` napr. `/foo/bar`, MacOs `/foo/bar/`, tj pridava lomitko.
V kode to nevadi, ked sa generuje cesta cez `filepath/filesystem.Join(...)`, ale tento test neprechadzal.
Rovno s Win, budeme spustat testy aj na MacOs, kedze je to pridanie jedneho riadku do `matrix`, a jedine tento test by nepresiel.